src/custom/CortiAuth.ts

```diff
@@ -74,6 +83,8 @@ export class Auth extends FernAuth {
     public async authorizeURL({
         clientId,
         redirectUri,
+        codeChallenge,
+        codeChallengeMethod,
```

Comment:
I remember from snippets you wrote that `codeChallengeMethod` was a constant. If that's correct, we don't need to ask users to pass it and we can add value ourselves. If it can be different -- we can anyway think about some default value (maybe not here, but when we apply query parameter)